docs/development/virtualizing_tails.rst

```diff
@@ -0,0 +1,244 @@
+
+.. code:: python
+
+   --- /usr/share/perl5/Tails/Persistence/Setup.pm	2017-06-30 09:56:25.000000000 +0000
+   +++ /usr/share/perl5/Tails/Persistence/Setup.pm.mod	2017-07-20 07:17:48.472000000 +0000
+   @@ -404,19 +404,6 @@
+
+        my @checks = (
+            {
+   -            method  => 'drive_is_connected_via_a_supported_interface',
+   -            message => $self->encoding->decode(gettext(
+   -                "Tails is running from non-USB / non-SDIO device %s.")),
+   -            needs_drive_arg => 1,
+   -        },
+   -        {
+   -            method  => 'drive_is_optical',
+   -            message => $self->encoding->decode(gettext(
+   -                "Device %s is optical.")),
+   -            must_be_false    => 1,
+   -            needs_drive_arg => 1,
+   -        },
+   -        {
+                method  => 'started_from_device_installed_with_tails_installer',
+                message => $self->encoding->decode(gettext(
+                    "Device %s was not created using Tails Installer.")),
+
+3. To apply the patch, from the Terminal run:
+
+.. code:: sh
+
+  sudo patch -p0 -d/ < persistence.patch
+
+4. Navigate to **Applications** then **Tails** and click **Configure
+   persistent volume**. Configure a persistent volume enabling all persistence
+   options.
+
+Shared Folders
+~~~~~~~~~~~~~~
+
+1. In the host OS, click on the Tails VM in VirtualBox and then go to
+   **Settings**.
+2. Click on **Shared Folders** and click the button on the right hand side to
+   add the folder. Navigate to the location of the SecureDrop repository on
+   your local machine. Check **Auto-mount**. Do not check
+   **Read-only**.
+
+3. Now reboot your Tails VM, decrypt the Persistent volume, and run the following
+   commands in a **Terminal**:
+
+.. code:: sh
+
+  mkdir ~/Persistent/securedrop
+  echo 'sudo mount -t vboxsf -o uid=$UID,gid=$(id -g) securedrop ~/Persistent/securedrop' >> /live/persistence/TailsData_unlocked/dotfiles/.bashrc
+
+Each time you open a Terminal in Tails with the Persistence volume decrypted,
+the shared folder will now automatically mount.
+
+Allow the Guest to Create Symlinks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, you'll need to allow the guest to create symlinks, which are
+`disabled by default in VirtualBox`_.
+
+.. _`disabled by default in VirtualBox`: https://www.virtualbox.org/ticket/10085#comment:12
+
+Shut down the Tails VM, and in your host run:
+
+.. code:: sh
+
+  VBoxManage setextradata "Admin Workstation" VBoxInternal2/SharedFoldersEnableSymlinksCreate/securedrop 1
+
+.. note:: If you named your Tails VM something other than "Admin Workstation",
+    you can run ``VBoxManage list vms`` to get the name of the Virtual Machine.
+
+Finally, restart VirtualBox.
+
+Configure Networking
+~~~~~~~~~~~~~~~~~~~~
+
+In order to communicate with the server VMs, you'll need to attach this
+virtualized *Admin Workstation* to the ``securedrop`` network.
```

Comment:
_No changes requested_, just FYI:  _technically_ the internal network will be named after whatever the repo for securedrop is named on the _Developer Workstation_ (running macOS in this case). That's configured via the `internal_network_name` helper function in the `Vagrantfile`, and was intended to support multiple concurrent VM setups on the same workstation (#1069).

Reply:
Good call, added a note about this in the text